src/display/api.js

```diff
@@ -1514,7 +1523,9 @@ class PDFPageProxy {
       operatorList: intentState.operatorList,
       pageIndex: this._pageIndex,
       canvasFactory: canvasFactory || this._transport.canvasFactory,
+      filterFactory: filterFactory || this._transport.filterFactory,
```

Comment:
Given the comments above:
```suggestion
      filterFactory: this._transport.filterFactory,
```